scripts/aptly-repo-from-image-diff.sh

```diff
@@ -100,14 +100,15 @@ aptly repo search image-a | xargs aptly repo copy image-a upgrade-repository ||
 
 #
 # Here we're performing step 3 from the comment above, but since the
-# "delphix-upgrade-verification" package is a bit different than most
-# other packages on a Delphix appliance, we need to handle that package
-# uniquely here. Specifically, we want to ensure this package is always
-# contained in the resultant aptly repository (even if the package is
-# the same within both image A and image B).
+# "delphix-upgrade-verification" and "delphix-entire" packages are a bit
+# different than most other packages on a Delphix appliance, we need to
+# handle these packages uniquely here. Specifically, we want to ensure
+# these packages are always contained in the resultant aptly repository
+# (even if the package is the same within both image A and image B).
```

Comment:
I remember reading elsewhere why we have delphix-entire package in the first place. Could we replicate that reasoning here again (or reference where one might learn more) so that if a developer is introduced to delphix-entire here they are not left wondering why they are different or worse just trusting this comment and moving on?

Reply:
I don't think here is the right place to explain what the delphix-entire package is, and/or why it's different.. similarly, I don't think it's the right place to explain what the delphix-upgrade-verification package is, and/or why it's different.

I do think it'd be reasonable to drop a reference to some other location where we explain that.. I'm not sure where to refer the read to, though. Do you have an idea of where we'd refer them to? I don't think we have either of those packages documented in a place that'd be easy to refer to...

Reply:
I think it might be reasonable to touch on why we always want these packages to remain in the sparse image, though.. I could expand a bit on that, specifically.

Reply:
I just added some more commentary, let me know if you think that's sufficient (or not).